File: Atividades/Reforço de Lógica/PHP/EX06.php
<?php 
// Recebendo até quando deve contar:
$contador = readline("Até quanto eu devo contar? ");
echo "\n";
// Adiciona 1 ao contador para contar até o exato número solicitado
$contador++;

// Loop para contar até o número solicitado, de 1 em 1:
for($i = 1; $i < $contador; $i++){
echo "$i\n";
// Espera um segundo:
sleep(1);
}

echo "\n---------------------FIM!---------------------";
?>

File: Atividades/Reforço de Lógica/Python/EX06.py
# ------------------- Contadora (conta até um número) ------------------- #

from time import sleep # Importando a biblioteca da função sleep para esperar um segundo entre as linhas de código

contador = int(input("Até quanto eu devo contar? ")) # Pedindo até que número ele deve contar
print("\n")

for i in range(contador): # Percorre um loop cuja duração é o número que o usuário deu
    print(f"{i}\n") # Printa o i (que será o número atual sendo percorrido no loop) e pula uma linha
    sleep(1) # Espera 1 segundo antes de repetir o loop

print("\n----------------------------FIM!----------------------------")